src/snap.py

```diff
@@ -55,6 +55,8 @@ def install(self) -> bool:
                 ]
             ]
 
+            snap.hold_refresh(forever=True)
```

Comment:
See comments on Kafka charm too -- this is too far reaching. We need to hold *just* the zookeeper snap.